src/snowflake/snowpark/modin/plugin/docstrings/dataframe.py

```diff
@@ -3168,6 +3168,7 @@ def stack():
         Parameters
         ----------
         level : int, str, list, default -1
+            level != -1 and MultiIndex dataframes are not yet supported by Snowpark pandas.
```

Comment:
Oh I would actually add a Notes section where we specify this - besides that LGTM!

Reply:
Oh ok, yeah will change this